infisical.js

```diff
@@ -133,14 +144,14 @@ export const awsIamLogin = async ({ identityId, domain }) => {
   }
 };
 
-const getAwsRegion = async () => {
+const getAwsRegion = async (axiosInstance) => {
   const region = process.env.AWS_REGION; // Typically found in lambda runtime environment
   if (region) {
     return region;
   }
 
   try {
-    const tokenResponse = await axios.put(
+    const tokenResponse = await axiosInstance.put(
```

Comment:
wouldn't these be messed up because you now pass in the "domain" value?

<img width="895" height="263" alt="image" src="https://github.com/user-attachments/assets/4e82c640-1502-491d-93fe-1f8598866692" />